### PR TITLE
Support KUBE_CONFIG_DATA

### LINKED ISSUE #115: Support KUBE_CONFIG_DATA
Support a complete /tmp/.kube/config file where specified.

This would help where we have this specified already in CI and provide a useful config option?

### PR DESCRIPTION
**Pull request by lewismarshall**
Fixes #115

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -828,6 +842,27 @@ func createCertificateAuthority(path, content string) error {
...
+// getKdTmpDir will get (or create a TempDir the first time)
+func getKdTempDir() string {
+	if len(tmpDir) < 1 {
+		// Update the global var used for cleanup
+		tmpDir, _ = ioutil.TempDir("", "kd")
+	}
+	return tmpDir
+}
+
+// createCertificateAuthority creates if required a certificate-authority file
```
Comment (marcinc) [inline main.go]: wrong comment.

File: README.md [hunk 2]
```diff
@@ -312,7 +312,9 @@ GLOBAL OPTIONS:
    --debug                                debug output [$DEBUG, $PLUGIN_DEBUG]
    --debug-templates                      debug template output [$DEBUG_TEMPLATES, $PLUGIN_DEBUG_TEMPLATES]
    --dryrun                               if true, kd will exit prior to deployment [$DRY_RUN]
+   --delete                               instead of applying the resources we are deleting them
```
Comment (marcinc) [inline README.md]: this should be really a separate commit :)

File: main.go (hunk 1 above)
Comment (lewismarshall) [reply main.go]: will push fix

File: README.md (hunk 2 above)
Comment (lewismarshall) [reply README.md]: yep - I guess :)
